wal/wal.go

```diff
@@ -886,11 +886,13 @@ func (w *WAL) seq() uint64 {
 	return seq
 }
 
-func closeAll(rcs ...io.ReadCloser) error {
+func closeAll(lg *zap.Logger, rcs ...io.ReadCloser) error {
+	var errRet error
```

Comment:
can we expose all errors? (collect them in slice, and return aggregated error at the end)?

Reply:
As I noted in the description, the return value is ignored by callers.

If you still feel aggregation is needed, @gyuho can you point me to an example where error slice is aggregated ?
I found errsToError() but it is in functional/tester/utils.go

Thanks

Reply:
@gyuho 
Waiting for your comment.

Reply:
I was thinking just aggregate as slice of strings, and join in return?

Reply:
`multierr` looks good as well :)

Reply:
@gyuho 
I adopted your first suggestion.

Please take a look.